internal/biz/resources/resources.go

```diff
@@ -174,7 +174,12 @@ func (uc *Usecase) Delete(ctx context.Context, id model.ReporterResourceId) erro
 		}
 	}
 
-	// TODO: delete the workspace tuple
+	if uc.Authz != nil {
+		_, err := uc.Authz.UnsetWorkspace(ctx, id.LocalResourceId, uc.Namespace, id.ResourceType)
```

Comment:
I believe we should be pointing at the internal biz function that was created

Reply:
+1, i think something more like
```go
	if uc.Authz != nil {
		err := biz.DefaultUnsetWorkspace(ctx, id.LocalResourceId, uc.Namespace, id.ResourceType, uc.Authz)
		if err != nil {
			return err
		}
	}
```
to match other calls, seems like everything else calls DefaultWorkspace

Reply:
I fixed this. Thank you for catching it!